MAUI/MauiAppBasic/Views/UserView.xaml.cs

```diff
@@ -0,0 +1,60 @@
+using MauiAppBasic.MSALClient;
+using MauiAppBasic.ViewModels;
+using Microsoft.Graph;
+using Microsoft.Identity.Client;
+using System.Net.Http.Headers;
+
+namespace MauiAppBasic.Views;
+
+public partial class UserView : ContentPage
+{
+    private UserViewModel _userViewModel = new UserViewModel();
+    public UserView()
+    {
+        BindingContext = _userViewModel;
+        _ = GetUserInformationAsync();
+
+        InitializeComponent();
+    }
+
+    private async Task GetUserInformationAsync()
+    {
+        try
+        {
+            // call Web API to get the data
+            AuthenticationResult result = await PCAWrapper.Instance.AcquireTokenSilentAsync(AppConstants.Scopes);
+
+            HttpClient client = new HttpClient();
+            client.DefaultRequestHeaders.Authorization = new AuthenticationHeaderValue("Bearer", result.AccessToken);
+
+            GraphServiceClient graphServiceClient = new GraphServiceClient(client);
```

Comment:
While this is correct, we were advised by the SDK team to use this format 

[https://github.com/Azure-Samples/ms-identity-netcore-winui/blob/17ff68ee893bf3229e390418e67ba18726a95aa3/WinUIMSALApp/MainWindow.xaml.cs#L160](https://github.com/Azure-Samples/ms-identity-netcore-winui/blob/17ff68ee893bf3229e390418e67ba18726a95aa3/WinUIMSALApp/MainWindow.xaml.cs#L160)